experimental/fRIP/fRIP_MPRNA_LibraryWorkupProtocol.md

```diff
@@ -0,0 +1,115 @@
+**Protocol typed by Evan Hass**  
+**Last modified on 2020-05-05**
+
+# DNase treatment of RNA  
+1. Aliquot 200 ng of RNA from each sample into PCR tubes, and bring to 16 uL with nuclease-free water. If a given sample is too dilute for 16 uL to contain 200 ng of RNA, just use 16 uL.  
+2. Make DNase master mix (1x recipe):  
+
+- 2 uL 10x Worthington DNase I Buffer  
+- 2 uL Worthington DNase I (Fisher NC9878502)  
+
+3. Add 4 uL of DNase master mix to each PCR tube  
+4. Incubate reactions at 37°C for 30 minutes  
+5. Stop reactions by incubating at 75°C for 10 minutes  
+
+# cDNA Synthesis  
+**_NOTE: All reagents for this cDNA synthesis protocol are included in the SuperScript IV First-Strand Synthesis System (Thermo Fisher 18091200)._**  
+
+1. Transfer 10 uL of DNase-treated RNA to new PCR tubes, and save the other 10 uL  
+2. Make priming master mix (1x recipe):  
+
+- 1 uL nuclease-free water  
+- 1 uL 50 ng/uL Random Hexamers  
```

Comment:
Random Hexamer catalog, dntp mix Superscirpt -- each reagent should have the company it is used from or how it's made -- just extra careful detail -- sorry !

Reply:
At the beginning of the cDNA Synthesis section, there's a note that all these reagents are included in the SuperScript IV kit.